website/docs/customize/blueprints/index.mdx

```diff
@@ -31,16 +48,22 @@ Additionally, all blueprints have access to built-in context variables:
 - `goauthentik.io/enterprise/licensed`: Boolean indicating if an enterprise license is active
 - `goauthentik.io/rbac/models`: Dictionary of available RBAC models
 
-These can be accessed using the `!Context` tag, for example:
+When you create or edit a blueprint instance, these context variables can be accessed using the `!Context` tag in the YAML file (or in th e UI under **Advanced settings** on the Create/Edit page).
+
+**Example**:
 
 ```yaml
 conditions:
     - !Context goauthentik.io/enterprise/licensed
 ```
 
-## Storage - File
+## Blueprint storage
+
+When you add a new blueprint instance, you can define where the `.yaml` file is stored: as a local file, in an OCI registry, or internally in authentik's database.
```

Comment:
revert, reads as if authentik writes to the selected storage type; it just reads yaml data from there